src/resolvers/projectResolver.ts

```diff
@@ -988,8 +976,19 @@ export class ProjectResolver {
 
     await pubSub.publish('NOTIFICATIONS', payload);
 
-    if (config.get('TRIGGER_BUILD_ON_NEW_PROJECT') === 'true')
+    if (config.get('TRIGGER_BUILD_ON_NEW_PROJECT') === 'true') {
+      // TODO whats this? I think we can delete it
```

Comment:
Yes please delete this. This is netflify integration, when we had gatsby in the frontend, this triggered a full rerender of the app, super inneficient and we scrapped that after our rewrites of the frontend.

Reply:
Thanks @CarlosQ96  , I removed al the thing related to netlify